src/Sylius/Bundle/CoreBundle/Fixture/OrderFixture.php

```diff
@@ -113,6 +119,7 @@ protected function configureOptionsNode(ArrayNodeDefinition $optionsNode): void
                 ->scalarNode('channel')->cannotBeEmpty()->end()
                 ->scalarNode('customer')->cannotBeEmpty()->end()
                 ->scalarNode('country')->cannotBeEmpty()->end()
+                ->booleanNode('fulfillOrder')->defaultFalse()->end()
```

Comment:
Use snake case here
And maybe you should add percentage value here instead of boolean (sth like `percentage_of_fulfilled_orders`), because in the current implementation, the information `fulfillOrder: true` is not 💯% true

Reply:
I thought about it, but @pamil suggested that boolean flag will be enough. I can change it or change only name like `fulfill_majority_orders` ?